NEWS.md

```diff
@@ -1,16 +1,34 @@
-# SIMplyBee version 0.3.0 #
+# SIMplyBee version 0.4.0 #
```

Comment:
Why is this the 0.4.0 now?

Reply:
I have setup NEWS for the new version already;)